UefiTestingPkg/FunctionalSystemTests/MpManagement/Driver/AARCH64/AsmSupport.S

```diff
@@ -55,9 +55,10 @@ ASM_PFX(AsmApEntryPoint):
 
 JumpToCFunction:
 // Calculate stack address
-  // x2 contains the index for the current processor plus 1
+  // x2 contains the index for the current processor

```

Comment:
might be good to improve this comment to explain the +1 part, or add a comment below, because the current comment implies the x2 is already the index so the +1 later is perplexing.

Reply:
Will do, thanks!

Reply:
Updated.